### PR TITLE
Fixed phpdoc

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Model/Dashboard.php [hunk 1]
```diff
@@ -18,16 +18,39 @@
...
+     */
     protected $name;
 
+    /**
+     * @var bool
+     */
     protected $enabled;
 
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Model/Dashboard.php]: `DashboardBlockInterface[]`

**Comment by core23** [comment]
Fixed @Soullivaneuh

File: Entity/BlockInteractor.php [hunk 2]
```diff
@@ -23,6 +24,9 @@
  */
 class BlockInteractor implements BlockInteractorInterface
 {
+    /**
+     * @var array
```
Comment (Soullivaneuh) [inline Entity/BlockInteractor.php]: `BlockInterface[]`.

From `\Sonata\BlockBundle\Model\BlockInterface`.

Comment (core23) [inline Entity/BlockInteractor.php]: Should be a boolean array.

**Comment by Soullivaneuh** [comment]
@core23 please rebase to make travis green.

**Comment by qsomazzi** [comment]
:+1:

**Pull request merged**